### ISSUE TITLE
Info object disappears after PowerByBand 

### ISSUE THREAD

**Issue by bacpac** [neurosity/eeg-pipes#38]
It seems some pipes, like `powerByBand()`, drop the `info` object with metadata which is available at start in the buffer.

You can follow the data in this example:
- `epoch` and `fft` keep the `info` object.
- `powerByBand` removes it (possibly caused by the reducer?)
```
        const eeg = fromEvent(self.eventEmitter, 'server:bci-file:data');

        const stream = eeg.pipe(
            epoch({ duration: 256, interval: 100, samplingRate: 250  }),
            fft({ bins: 256 }), // info object with timestamp data is still here
            powerByBand() // after this pipe, the info is gone :(
            ).subscribe((buffer) => {
                console.log(buffer)
            });
       );
```

I could use an `addInfo()` after the `powerByBand()` pipe, but I do not have access to that information anymore.

Is this by design, or possibly an extension that could be made to the functionality?

**PS:** Thank you for the awesome library, it already helped me a lot with my project.

**Comment by alexcastillo** [comment]
Thanks for reporting the issue, @bacpac!

I'll make sure to include a fix on the next version.

**Comment by alexcastillo** [comment]
This is now fixed in v4.3.0 by @yannickadam in PR #55. Please note the breaking change:

```
Moved the signalQuality object under info so it is preserved.
```
